Documentation/ReleaseNotes.md

```diff
@@ -1,9 +1,276 @@
 # Microsoft Mixed Reality Toolkit Release Notes
 
+- [Version 2.2.0](#version-220)
 - [Version 2.1.0](#version-210)
 - [Version 2.0.1](#version-201)
 - [Version 2.0.0](#version-200)
 
+## Version 2.2.0
+
+- [Upgrading projects](#upgrading-projects-to-220)
+- [What's new](#whats-new-in-220)
+- [Known issues](#known-issues-in-220)
+
+This release of the Microsoft Mixed Reality Toolkit supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+- (Experimental) Mobile AR
+  - iOS
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+NuGet requirements
+
+If importing the Mixed Reality Toolkit NuGet packages, the following software is recommended.
+
+- [NuGet for Unity](https://github.com/GlitchEnzo/NuGetForUnity)
+
+### Upgrading projects to 2.2.0
+
+The 2.2.0 release has some changes that may impact application projects. Breaking change details, including mitigation guidance, can be found in the [**Updating 2.1.0 to 2.2.0**](Updating.md#updating-210-to-220) article.
+
+**Updating using .unitypackage files**
```

Comment:
Is there any documentation that calls out how to go from .unitypackage to nuget?

Reply:
I wouldn't block on a release for this, but it seems like it may be worth trying to have all documentation possible to move folks towards using nuget.

Reply:
There is not, nor was there for 2.1.0.  Do you believe we need them for this release?